Autocoders/Python/src/fprime_ac/utils/version.py

```diff
@@ -0,0 +1,14 @@
+""" fprime version handling and reporting """
+import os
+from setuptools_scm import get_version
+
+ROOT_PARENT_COUNT=5
+
+def get_fprime_version():
+    """ Gets the fprime version using setuptools_scm  """
+    # First try to read the SCM version
+    try:
+        return get_version(root=os.sep.join([".."] * ROOT_PARENT_COUNT), relative_to=__file__)
```

Comment:
Optional nitpick: Does get_version need to run in the absolute repository root?  You might be able to get away with not needing to set root.

Reply:
yes it does.